app/routes/missions/gecam.md

```diff
@@ -2,3 +2,43 @@
 meta:
   title: GCN - GECAM
 ---
+
+# Gravitational Wave High-energy Electromagnetic Counterpart All-sky Monitor
+
+image here?
+
+**Launch Date:** December 9, 2020
```

Comment:
It launched 2 years ago? Why are we only adding them to GCN now?

Reply:
Yes, they launched 2 years ago.  I think it took a while for them to ask to be added, and adding them has taken some time too.